packages/guides/src/Handlers/ParseDirectoryHandler.php

```diff
@@ -98,14 +101,23 @@ private function getDirectoryIndexFile(
             $hashedContentFromFilesystem[$itemFromFilesystem['basename']] = true;
         }
 
-        foreach (self::INDEX_FILE_NAMES as $indexName) {
+        if ($indexName !== null) {
             $indexFilename = sprintf('%s.%s', $indexName, $extension);
             if (isset($hashedContentFromFilesystem[$indexFilename])) {
                 return $indexName;
             }
```

Comment:
If you extract this part to a method we reduce the duplicated lines. As you repeated this in the foreach loop.

Reply:
We only have one for each loop now